SCF/Assignment3C/TestArea.java

```diff
@@ -0,0 +1,133 @@
+import java.util.Scanner;
+
+/**
+ * 
+ * @author Nilesh Patel
+ * AreaMain Class used to perform test on Area Class
+ * It interact with user and take input and get the result 
+ * from Area Class and print to console
+ *
+ */
+public class TestArea 
+{
+
+	public static void main(String[] args) 
+	{
+		
+		Area area = new Area();
+		Scanner userInput = new Scanner(System.in);
+		
+		while(true)
+		{	
+			System.out.println("1. Get Area Of Right Angle Triangle");
+			System.out.println("2. Get Area Of Rectangle");
+			System.out.println("3. Get Area Of Square");
+			System.out.println("4. Get Area Of Circle");
+			System.out.println("5. Exit");
+			System.out.print("Please Select Opion : ");
+			int userChoice = userInput.nextInt();
+			
+			//loop termination logic
+			if(userChoice == 5)
+			{
+				System.out.println("Thank You");
+				System.exit(0);
+			}
+			
+			//switch case for option selection
+			switch(userChoice)
+			{
+				//for getting area of Right Angle Triangle
+				case(1):
+				{
+					try
+					{
+						System.out.print("Enter Height Of Right Angle Triangle : ");
+						double heightOfRightAngleTriangle = userInput.nextDouble();
+						System.out.print("Enter Base Of Right Angle Triangle : ");
+						double baseOfRightAngleTriangle = userInput.nextDouble();
+						
+						double areaOfRightAngleTriangle = area.getAreaOfTriangle(heightOfRightAngleTriangle, baseOfRightAngleTriangle);
+						System.out.printf("Area Of Right Angle Triangle is : %.2f \n", areaOfRightAngleTriangle);
+						
+					}
+					catch(Exception e)
+					{
+						System.out.println(e);
```

Comment:
Use ```e.printStackTrace()``` at every place for printing exception message.

Reply:
Done